### PR TITLE
8990: Bump up pa11y timeout for public search test, enable test

### PR DESCRIPTION
**Pull request by TomElliottFlexion**

### CODE REVIEW

File: web-client/pa11y/pa11y-public-user-search.js [hunk 1]
```diff
@@ -61,24 +61,24 @@ module.exports = [
...
+  },
+  {
+    actions: [
+      'wait for #tab-opinion to be visible',
+      'click element #tab-opinion',
+      'wait for #clear-search to be visible',
+      'click element #clear-search',
+      'wait for #opinion-search to be visible',
+      'set field #opinion-search to sunglasses',
+      'set field #startDate-date-start to 08/01/2001',
```
Comment (mrinsin) [inline web-client/pa11y/pa11y-public-user-search.js:79]: I think we will need to add a line or two to select 'custom date range' similar to the order tests! It's a new field that was added while these tests were commented out

**Comment by kkoskelin** [comment]
Frequently pa11y timeouts aren't about the endpoint not replying, but that the actions could not all be executed successfully.  You'll probably have to crawl over each one manually verifying selectors and so forth.

File: web-client/pa11y/pa11y-public-user-search.js (hunk 1 above)
Comment (TomElliottFlexion) [reply web-client/pa11y/pa11y-public-user-search.js:79]: Will do. These are the same failures that were occurring before this test was uncommented

**Review by mrinsin — APPROVED** [review]

File: web-client/pa11y/pa11y-public-user-search.js (hunk 1 above)
Comment (mrinsin) [reply web-client/pa11y/pa11y-public-user-search.js:79]: omg we are both so silly - this is for the external opinion search (and we have not worked on the story to enable it yet lol)

Comment (TomElliottFlexion) [reply web-client/pa11y/pa11y-public-user-search.js:79]: ...silly is generous, but I'll take it

**Pull request closed without merging**